ajax/libs/shopify-buy/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "shopify-buy",
+  "description": "The JS Buy SDK is a lightweight library that allows you to build ecommerce into any website. It is based on Shopify's API and provides the ability to retrieve products and collections from your shop, add products to a cart, and checkout.",
+  "filename": "shopify_buy.min.js",
+  "keywords": [
+    "shopping",
+    "cart",
+    "shopify"
+  ],
+  "author": "Shopify Inc.",
+  "homepage": "https://shopify.github.io/js-buy-sdk",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Shopify/js-buy-sdk"
+  },
+  "npmName": "shopify-buy",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
Let's use `shopify-buy*` instead. It can avoid our bot fetching files under `node-lib` in the older versions of the lib. Thanks!

The pattern should be enough for current file structure (at lease before v1). It seem to change in near future. We might need to pay attention it. cc @cdnjs/intern3

Reply:
@extend1994 Did you mean `shopify-buy/*` ?

Reply:
I think it's `shopify-buy*` not `shopify-buy/*`, `shopify-buy` is not a directory.